v2/api/apimanagement/v1api20220801/structure.txt

```diff
@@ -2,6 +2,133 @@
 github.com/Azure/azure-service-operator/v2/api/apimanagement/v1api20220801
 ├── APIVersion: Enum (1 value)
 │   └── "2022-08-01"
+├── Backend: Resource
+│   ├── Owner: Service
+│   ├── Spec: Object (11 properties)
+│   │   ├── AzureName: Validated<string> (2 rules)
+│   │   │   ├── Rule 0: MaxLength: 80
+│   │   │   └── Rule 1: MinLength: 1
+│   │   ├── Credentials: *Object (5 properties)
+│   │   │   ├── Authorization: *Object (2 properties)
+│   │   │   │   ├── Parameter: Validated<*string> (2 rules)
+│   │   │   │   │   ├── Rule 0: MaxLength: 300
+│   │   │   │   │   └── Rule 1: MinLength: 1
+│   │   │   │   └── Scheme: Validated<*string> (2 rules)
+│   │   │   │       ├── Rule 0: MaxLength: 100
+│   │   │   │       └── Rule 1: MinLength: 1
+│   │   │   ├── Certificate: Validated<string[]> (1 rule)
```

Comment:
I reckon `Certificate` should be marked as a secret along with a few other props under `Authorization`. Not sure if other properties would also contain sensitive auth information - need to investigate on that.